Update person records (#397)
MukurtuCMS/mukurtu-user-manual: docs/person-records/PersonRecords.md

Reviewer: ```suggestion
Enter the person's name as they should be primarily identified in the *Name* field. Names can be formatted as `Last, First, Middle`, `Last, First (Nickname)`, `Family name, Given name`, or any other format that is appropriate for the person.
```

Reviewer: ```suggestion
1. In the *Other Names* field, use the text box to select existing terms or add new ones. New names will be added to the People taxonomy.
```

Reviewer: ```suggestion
Media assets are often included in person records, though they are not required. Supported media types are images, documents, video, audio, and embed code. Person records can include more than one media asset, and each media asset can be a different media type. Media assets can be assigned a different cultural protocol from the person record to allow differential access to the media assets and metadata. For instructions on how to add a media asset, refer to the [Create Media Assets](../media/CreateMediaAssets.md) article.
```

### Related people

!!! requirement
    Interpersonal relationships are not automatically bi-directional, and must be added to both person records. 

> You can add a related person to a person record by adding an extant person record or by creating one on the fly. Related people is used to reflect real-world relationships between people. Examples include family, professional, cultural, educational, or any other kind of interpersonal relationships. 

Reviewer: ```suggestion
You can add a related person to a person record by selecting an existing person record or by creating one on the fly. Related people is used to reflect real-world relationships between people. Examples include family, professional, cultural, educational, or any other kind of interpersonal relationships. 
```

To add a related person, navigate to the **Related People** section and select the "Add Related Person" button. For information on creating a related person record on the fly, refer to the [Create a person record on the fly](#create-a-person-record-on-the-fly) section of this article. 

![Screenshot of the add related person button in the related people section](../_embeds/person6.png)

> #### Add an extant person record

Reviewer: ```suggestion
#### Connect an existing person record
```

1. Select the "Select person" button. 

    ![Screenshot of the related person field with the select content button highlighted.](../_embeds/person7.png)

>     !!! tip 

Reviewer: I don't think this is needed.

4. Select and drag the arrows to reorder your related people if necessary. To remove a related person, select the "Remove" button.

#### Create a person record on the fly

To create a person record on the fly, select the "Can't find the person? Create a new person record." button. 

![Screenshot of the related person selection field with the can't find the person create a new person record button highlighted.](../_embeds/person15.png)

This opens a new tab in your browser where you can create a person record. 

> - Create a person record as outlined in this article, then select the "Save" button. 

Reviewer: This needs information about the directionality since the on the fly record will be missing the relationship, right?
